metadata-models/src/main/pegasus/com/linkedin/notebook/TextCell.pdl

```diff
@@ -0,0 +1,14 @@
+namespace com.linkedin.notebook
+
+/**
+ * Text cell in a Notebook, which will present content in text format
+ */
+record TextCell includes CommonCellAttributes {
+  /**
+   * The actual text in a TextCell in a Notebook
+   */
+   @Searchable = {
+     "fieldType": "TEXT"
+   }
+  text: string
```

Comment:
Do we want to make this searchable? So basically index any text cells in the notebook?

Reply:
Right I totally understand the usefulness of making it searchable. My main concern is that the amount of text in the notebook can be unbounded.

Reply:
kk. I think it might be an issue if the notebook is widely used in a large scale company. Is there any way to make it non-searchable by default but we could make it searchable later if it makes sense to the org?

Reply:
Hmn not really. we don't have feature flags supported on annotations unfortunately

Reply:
kk. I am fine to remove it in this PR and enable searchable in my local repo

Reply:
SGTM